test/acceptance/behave/components/ngsiv2/entities/list_entity_by_id.feature

```diff
@@ -0,0 +1,893 @@
+      | "room9"   | [ "json", ["a", 34, "c", {"r": 4, "t":"4", "h":{"s":"3", "g":"v"}}], "of", 6] | list     |
+      | "room10"  | {"x": "x1","x2": "b"}                                                         | dict     |
+      | "room11"  | {"x": {"x1": "a","x2": "b"}}                                                  | dict     |
+      | "room12"  | {"a":{"b":{"c":{"d": {"e": {"f": "ert"}}}}}}                                  | dict     |
+      | "room13"  | {"x": ["a", 45.56, "rt"],"x2": "b"}                                           | dict     |
+      | "room14"  | {"x": [{"a":78, "b":"r"}, 45, "rt"],"x2": "b"}                                | dict     |
+      | "room15"  | "41.3763726, 2.1864475,14"                                                    | str      |
+      | "room16"  | "2017-06-17T07:21:24.238Z"                                                    | str      |
+      | "room17"  | null                                                                          | NoneType |
+
+  @without_metadata_type
+  Scenario Outline:  get an entity by ID in NGSI v2 without metadata type
+    Given  a definition of headers
+      | parameter          | value            |
+      | Fiware-Service     | test_id_metadata |
+      | Fiware-ServicePath | /test            |
+      | Content-Type       | application/json |
+    When create "1" entities with "3" attributes
+      | parameter        | value             |
+      | entities_type    | house             |
+      | entities_id      | room              |
+      | attributes_name  | temperature       |
+      | attributes_value | <attribute_value> |
+      | metadatas_number | 2                 |
+      | metadatas_name   | very_hot          |
+      | metadatas_value  | hot               |
+    And verify that receive several "Created" http code
+    When get an entity by ID "room"
+      | parameter | value         |
+      | attrs     | temperature_0 |
+    Then verify that receive an "OK" http code
+    And verify that the entity by ID is returned
+    And verify that the entity by ID is returned
+    Examples:
+      | attribute_value         |
+      | 017-06-17T07:21:24.238Z |
+      | 34                      |
+      | 34.4E-34                |
+      | temp.34                 |
+      | temp_34                 |
+      | temp-34                 |
+      | TEMP34                  |
+      | house_flat              |
+      | house.flat              |
+      | house-flat              |
+      | house@flat              |
+      | habitación              |
+      | españa                  |
+      | barça                   |
+      | random=10               |
+      | random=100              |
+      | random=1000             |
+      | random=10000            |
+      | random=100000           |
+
+  @compound_with_metadata_without_meta_type @BUG_1106 @skip
+  Scenario Outline: get an entity by ID in NGSI v2 with special attribute values and with metadatas but without metadata
```

Comment:
This is a typo, it is `"with metadatas but without metadata type"` instead of `"with metadatas but without metadata"`